src/app/_components/TherapiesSection.js

```diff
@@ -0,0 +1,46 @@
+import { TopWave } from './TopWave';
+import { BottomWave } from './BottomWave';
+import { TherapyCard } from './TherapyCard';
+import { Heading } from './Typography';
+
+export function TherapiesSection() {
+  return (
+    <section className="relative bg-primary-200 bg-blend-multiply ">
+      <div
+        style={{
+          position: 'absolute',
+          top: 0,
+          left: 0,
+          right: 0,
+          bottom: 0,
+          backgroundImage: "url('/assets/noise.png')",
+          backgroundSize: 'cover',
+          backgroundPosition: 'center',
+          opacity: 0.4,
+        }}
+      ></div>
```

Comment:
I believe you can use a self-closing tag here - <div style={...} />

The linter had to give you an error or needs to enable such a rule

Reply:
done, added